src/day8/re_frame/trace.cljs

```diff
@@ -259,7 +305,26 @@
       (let [new-panel (.createElement js/document "div")]
         (.setAttribute new-panel "id" id)
         (.appendChild (.-body js/document) new-panel)
+        (js/window.focus new-panel)
         new-panel))))
 
+(defn inject-styles []
```

Comment:
I haven't looked too closely, but could https://google.github.io/closure-library/api/goog.style.html#installSafeStyleSheet be helpful here?